src/mjolnir/graphenhancer.cc

```diff
@@ -1219,8 +1219,8 @@ void ProcessEdgeTransitions(const uint32_t idx,
     uint32_t left_count = 0;
     if (ntrans > 2) {
       for (uint32_t j = 0; j < ntrans; ++j) {
-        // Skip the from and to edges
-        if (j == i || j == idx) {
+        // Skip the from and to edges; also skip roads under construction
+        if (j == i || j == idx || edges[j].use() == Use::kConstruction) {
```

Comment:
@genadz can you just explain this a little bit? when we look up stop impact later on, dont we do that by local index? if we skip edges under construction, do we get an off by one index problem when we look up the stop impact later?

Reply:
this impacts only `left_count` and `right_count` variables that are used to set `edge_to_left` and `edge_to_right` flags. So, this change says even if there is a road under construction between two real edges, it will not be considered to calculate `edge_to_left` and `edge_to_right` flags